.eslintrc

```diff
@@ -3,10 +3,24 @@
   "root": true,
   "overrides": [
     {
-      "files": ["packages/grafana-ui/**/*/!(*.story).{ts,tsx}", "public/app/**/*.{ts,tsx}"],
+      "files": [
+        "packages/grafana-ui/**/*/!(*.story).{ts,tsx}",
```

Comment:
Shouldn't this glob be filtering out stories in `packages/grafana-ui`? 🤔

Reply:
Indeed, that's why there are fixes to the stories in this PR :)

Reply:
Ha. I'm reading it backwards! Carry on. 🤪